awx/ui_next/src/screens/Organization/OrganizationDetail/OrganizationDetail.jsx

```diff
@@ -97,7 +97,13 @@ function OrganizationDetail({ i18n, organization }) {
         {summary_fields?.default_environment?.name && (
           <Detail
             label={i18n._(t`Default Execution Environment`)}
-            value={summary_fields.default_environment.name}
+            value={
+              <Link
+                to={`/execution_environments/${summary_fields.default_environment.id}/details`}
```

Comment:
```suggestion
                to={`/execution_environments/${summary_fields?.default_environment?.id}/details`}
```

Good rule of thumb: Don't trust summary fields

Reply:
I will update as per your suggestions. I was relying on the first if 

```
{summary_fields?.default_environment?.name ...

``` 
If that is the true, then we can proceed. Thanks!

Reply:
I see it now! yep :+1: